src/Host/Client/Test/DataInspection/ValuesTest.cs

```diff
@@ -31,13 +31,14 @@ public class ValuesTest : IAsyncLifetime {
         private const REvaluationResultProperties AllFields = unchecked((REvaluationResultProperties)~0);
 
         private readonly MethodInfo _testMethod;
+        private readonly IRHostBrokerConnector _brokerConnector = new RHostBrokerConnector(name: nameof(ValuesTest));
```

Comment:
Order is actually explicit for instance fields as well (it's always textual order, since they cannot depend on each other). I didn't know about the stack trace, though. Will do.

---
In reply to: [75027791](https://github.com/Microsoft/RTVS/pull/2173#discussion_r75027791) [](ancestors = 75027791,75011847,74972244)